energy-growers/src/components/AccordionList/Accordion/Accordion.scss

```
@@ -0,0 +1,47 @@
+@import "../../../assets/styles/globals.scss";
+
+.eg-container-accordion {
+  border-bottom: rem(1) solid map-get($colors, light-grey);
+
+  .eg-accordion-item-container {
+    overflow: hidden;
+    transition: all 0.3s ease-in-out;
+  }
+  .eg-content-accordion {
+    @include remove-button-background;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin: rem(24) rem(24) rem(24) rem(0);
+    .eg-question-accordion {
+      color: map-get($colors, dark-aqua);
+      text-align: start;
+      font-weight: 700;
+    }
+    .icon-arrow {
+      color: map-get($colors, dark-blue);
+      transform: rotate(90deg);
+      margin-left: rem(13);
+      width: rem(20.541);
+      height: rem(20);
+      font-weight: 700;
```

```suggestion
      @include font-size-line-weight(20, 20, 700);
      
      &.eg-rotate-arrow {
        transform: rotate(-90deg);
        transition: all 0.3 ease-in-out;
      }
```

la mixin le creiamo per utilizzarle, margine inutile

le classi vanno annidate bene... NO IMPORTANT